include/SFML/Window/Event.hpp

```diff
@@ -277,42 +277,48 @@ class SFML_WINDOW_API Event
     ////////////////////////////////////////////////////////////
     /// \brief Default constructor
     ///
-    /// Sets the event to sf::Event::Empty
+    /// Sets the event subtype to `sf::Event::Empty`
     ///
     ////////////////////////////////////////////////////////////
     Event() = default;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Construct from a given sf::Event subtype
+    /// \brief Construct from a given `sf::Event` subtype
     ///
-    /// \param t Event subtype
+    /// \tparam EventSubType Type of event subtype used to construct the event
```

Comment:
```suggestion
    /// \tparam EventSubtype Type of event subtype used to construct the event
```
`EventSubType` -> `EventSubtype`. "Subtype" is one word so I think the "t" in "type" does not deserve to be capitalized.

I'm also not in love with the "type of event subtype" phrasing. I think we can just say 
```
/// \tparam EventSubtype Event subtype used to construct the event
```

Reply:
As a template type it should be prefixes with T: TEventSubtype

Reply:
We don't follow that convention anywhere else -- if you feel strongly about that naming convention I can apply it here and then create a separate PR to apply it to the rest of the library. What do you decide @eXpl0it3r?

Reply:
@ChrisThrasher:

> EventSubType -> EventSubtype. "Subtype" is one word so I think the "t" in "type" does not deserve to be capitalized.

Good point, will fix.

> I'm also not in love with the "type of event subtype" phrasing. I think we can just say [...]

I'm not in love with it either, but I'd prefer keeping it. Consider

```cpp
/// \tparam EventSubtype Type of event subtype used to construct the event
///
/// \param eventSubtype Event subtype used to construct the event
```

Our meaning of "event subtype" here is not the type (for example) `sf::Event::Closed`, but it is the actual object of type `sf::Event::Closed`. So our meaning of "event subtype" is "object of type `sf::Event::*`. Therefore to differentiate `eventSubtype` (object) and `EventSubtype` (type) it is more correct to say *"type of event subtype"*, despite the cacophony.

Reply:
The only place where I found that we don't just use `T` or `U` is in the new `Time` API.

```
    template <typename Rep, typename Period>
    constexpr Time(const std::chrono::duration<Rep, Period>& duration);
```

Yes, I think it makes sense to prefix them with `T`. Also give `Rep` a more speaking name? I'd have to look up `std::chrono` just to understand what type is expected here.